Makefile

```diff
@@ -76,7 +76,11 @@ bench:
 # Creates a release build in a containerized build environment of the static library for Alpine Linux (.a)
 release-build-alpine:
 	# build the muslc *.a file
-	docker run --rm -v $(shell pwd)/libwasmvm:/code $(BUILDERS_PREFIX)-alpine
+	docker run --rm -v $(shell pwd)/libwasmvm:/code \
+		-e CARGO_TARGET_AARCH64_UNKNOWN_LINUX_MUSL_LINKER=aarch64-linux-musl-gcc \
+		-e CC_aarch64_unknown_linux_musl=aarch64-linux-musl-gcc \
+		-e CFLAGS_aarch64_unknown_linux_musl="" \
```

Comment:
We typically set those things inside of the builder (the script or a config file). Do we need to pass it as an argument here?

Reply:
Good catch! That's a leftover from testing. I think it's not even needed.

Reply:
Fixed!